src/AzureAuth/Program.cs

```diff
@@ -103,7 +103,7 @@ private static void Main(string[] args)
                 Environment.Exit(2);
             };
 
-            new Lasso(app, options, services).Execute(args);
+            new LassoMcMaster(app, options, services).Execute(args);

```

Comment:
I assume you tested this manually after updating it yourself? You didn't observe anything else that requires changing between these releases?